.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -621,7 +626,27 @@ def write(x=''):
 
 ##############################################################################
 # Write submitty json
+user_id_requirements = {
+    "any_user_id": True,
+    "require_name": False,
+    "min_length": 6,
+    "max_length": 25,
+    "name_requirements": {
+        "given_first": False,
+        "given_name": 2,
+        "family_name": 4
+    },
+    "require_email": False,
+    "email_requirements": {
```

Comment:
similarly can we add some comments / documentation in the code.  Or alternately, put the link to the documentation page (I know you have a PR for documentation, add the link to where that documentation will be when merged)